src/transformers/visitors/checks/incrementedVisitor.ts

```diff
@@ -334,6 +334,7 @@ export default {
       const assignmentOp = parentExpressionStatement?.node.expression?.operator;
       // if we don't have a parent expression or that expression can't hold an incrementation, we exit
       if (!lhsNode) return;
+      if (!binOpToIncrements(path, state)?.operands) return;
 
       const { operands, precedingOperator } = binOpToIncrements(path, state);
```

Comment:
This fixes an error where the expression had no operands so `ts` would throw (correctly). e.g. the expression `a = a + b + c` has operands `b` and `c` but the expression `a = b*c` has no operands to do with incrementation, so we exit.